clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -1703,9 +1704,9 @@ static ArrayRef<options::ID> getUnsupportedOpts() {
       options::OPT_fprofile_instr_generate,
       options::OPT_fprofile_instr_generate_EQ,
       options::OPT_fprofile_arcs,
-      options::OPT_fno_profile_arcs, // -f[no-]profile-arcs
+      options::OPT_fno_profile_arcs,           // -f[no-]profile-arcs
       options::OPT_fno_profile_instr_generate, // -f[no-]profile-instr-generate
-      options::OPT_fcreate_profile,  // -fcreate-profile
+      options::OPT_fcreate_profile,            // -fcreate-profile
```

Comment:
Ordering here looks different than what is in llorg - `options::OPT_fno_profile_instr_generate` should fall after `options::OPT_fprofile_instr_generate_EQ`